src/goto-instrument/contracts/contracts.cpp

```diff
@@ -1260,3 +1258,63 @@ bool code_contractst::enforce_contracts(const std::set<std::string> &to_enforce)
   }
   return fail;
 }
+
+void code_contractst::find_static_locals(
+  const goto_functiont &function,
+  assigns_clauset &assigns)
+{
+  find_symbols_sett tmp_set;
```

Comment:
nit: we could call it `static_local_set` instead of `tmp_set`.

Reply:
the contents of this set only gets filtered in a second step